### PR TITLE
write API docs

### PR DESCRIPTION
**Pull request by nguyenvukhang**
docs are in `/apps/docs`

### CODE REVIEW

File: apps/docs/components/Parameter.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        dangerouslySetInnerHTML={{ __html: parseMarkdown(props.description) }}
+      />
+    </div>
+  )
+}
+
+const name: any = {
+  fontWeight: 'bold',
+  marginRight: '8px',
+}
```
Comment (weiseng18) [inline apps/docs/components/Parameter.tsx:39]: mb pls help me use tailwind

File: apps/docs/components/ParameterList.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  function has(arr: any[]) {
+    if (!arr) return false
+    if (arr.length === 0) return false
+    return true
+  }
+
+  return (
+    <div>
+      <h3 className="mt-0 mb-0">Parameters</h3>
```
Comment (weiseng18) [inline apps/docs/components/ParameterList.tsx:29]: my-0?

File: apps/docs/components/Response.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react'
+import Prism from 'prismjs'
+
+enum ResponseState {
+  fulfilled = 'fulfilled',
+  schema = 'schema',
+  // rejected = 'rejected',
```
Comment (weiseng18) [inline apps/docs/components/Response.tsx:7]: can remove since we won't be using

File: apps/docs/data/module/get.ts [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        'the context of the Euclidean spaces R^n.  Proofs of results will ' +
+        'be presented in the concrete setting.  Students are expected to ' +
+        'acquire computational facilities and geometric intuition with ' +
+        'regard to vectors and matrices.  Some applications will be ' +
+        'presented. Major topics: Systems of linear equations, matrices, ' +
+        'determinants, Euclidean spaces, linear combinations and linear ' +
+        'span, subspaces, linear independence, bases and dimension, rank of ' +
+        'a matrix, inner products, eigenvalues and eigenvectors, ' +
+        'diagonalization, linear transformations between Euclidean spaces, ' +
+        'applications.',
```
Comment (weiseng18) [inline apps/docs/data/module/get.ts:34]: looks like ma2001 description

File: apps/docs/components/ParameterList.tsx (hunk 2 above)
Comment (nguyenvukhang) [reply apps/docs/components/ParameterList.tsx:29]: fixed

File: apps/docs/components/Parameter.tsx (hunk 1 above)
Comment (nguyenvukhang) [reply apps/docs/components/Parameter.tsx:39]: fixed

File: apps/docs/components/Response.tsx (hunk 3 above)
Comment (nguyenvukhang) [reply apps/docs/components/Response.tsx:7]: fixed

File: apps/docs/data/module/get.ts (hunk 4 above)
Comment (nguyenvukhang) [reply apps/docs/data/module/get.ts:34]: close enough

**Pull request merged**